### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the important note as it creates ambiguity when saying M365 admin center as users dont have access to it.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7262

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Follow the steps below if you want to set user passwords to expire after a speci
 1. In the second box type when users are notified that their password will expire, and then select **Save**. Choose a number of days from 1 to 30.
 
 > [!IMPORTANT]
-> Password expiration notifications are no longer supported in the Microsoft 365 admin center or any Office apps.
+> Password expiration notifications are no longer supported in the Office web apps,  or [Office portal](https://portal.office.com).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
> Password expiration notifications are no longer supported in Office web apps or the [Office portal](https://portal.office.com).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Follow the steps below if you want to set user passwords to expire after a speci
 1. In the second box type when users are notified that their password will expire, and then select **Save**. Choose a number of days from 1 to 30.
 
 > [!IMPORTANT]
-> Password expiration notifications are no longer supported in the Microsoft 365 admin center or any Office apps.
+> Password expiration notifications are no longer supported in Office web apps or the [Office portal](https://portal.office.com).
```
Comment (tiburd) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
> Password expiration notifications are no longer supported in Office web apps or the [admin center](https://portal.office.com).
```

**Pull request merged**